### PR TITLE
Add codeclimate/eslint and fix issues

### PR DESCRIPTION
**Pull request by ghost**
Add codeclimate/eslint and fix issues

### CODE REVIEW

File: src/deep-root-angular/frontend/js/lib/angular-mocks.js [hunk 1]
```diff
@@ -794,56 +794,56 @@ angular.mock.animate = angular.module('ngAnimateMock', ['ng'])
...
-          if (!somethingFlushed) {
-            throw new Error('No pending animations ready to be closed or flushed');
-          }
-
-          $rootScope.$digest();
-        }
-      };
-
-      angular.forEach(
+                              var animate = {
```
Comment (vcernomschi) [inline src/deep-root-angular/frontend/js/lib/angular-mocks.js]: Please fix intend.

File: src/deep-root-angular/frontend/js/lib/system.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
....address))}function u(){d.detachEvent?d.detachEvent('onreadystatechange',i):(d.removeEventListener('load',i,!1),d.removeEventListener('error',l,!1)),n.removeChild(d)}var d=document.createElement('script');d.async=!0,d.attachEvent?d.attachEvent('onreadystatechange',i):(d.addEventListener('load',i,!1),d.addEventListener('error',l,!1)),a=e.System,e.System=s,d.src=o.address,o.metadata.integrity&&d.setAttribute('integrity',o.metadata.integrity),n.appendChild(d)}):r.call(this,o)}})}(),function(){function t(e,t,n){if(g=!0,t)t=(e.normalizeSync||e.normalize).call(e,t),n.name=t,t in e.defined||(e.defined[t]=n);else{if(h)throw new TypeError('Invalid anonymous System.register module load. If loading a single module, ensure anonymous System.register is loaded via System.import. If loading a bundle, ensure all the System.register calls are named.');h=n}}function n(e,t,a){if(a[e.groupIndex]=a[e.groupIndex]||[],-1==S.call(a[e.groupIndex],e)){a[e.groupIndex].push(e);for(var r=0,o=e.normalizedDeps.length;o>r;r++){var s=e.normalizedDeps[r],i=t.defined[s];if(i&&!i.evaluated){var l=e.groupIndex+(i.declarative!=e.declarative);if(void 0===i.groupIndex||i.groupIndex<l){if(void 0!==i.groupIndex&&(a[i.group
```
Comment (vcernomschi) [inline src/deep-root-angular/frontend/js/lib/system.js]: Please undo changes in `src/deep-root-angular/frontend/js/lib/` folder

**Comment by vcernomschi** [comment]
Should be tested on local.

**Comment by devs-deep** [comment]
:white_check_mark: coverage increased from 0% to 100%

File: README.md [hunk 3]
```diff
@@ -33,26 +33,26 @@ inside Git Bash.
 ### Step 2. Install Microservice(s) Locally
 
 ```bash
-deepify install github://MitocGroup/deep-microservices-todomvc ~/deep-microservices-todomvc
+deepify install github://MitocGroup/deep-microservices-root-angular ~/deep-microservices-root-angular
```
Comment (vcernomschi) [inline README.md]: Please update only badge in README.

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+  init-declarations: 0
+  no-catch-shadow: 2
+  no-delete-var: 2
+  no-label-var: 2
+  no-shadow-restricted-names: 2
+  no-shadow: 0
+  no-undef-init: 2
+  no-undef: 2
+  no-undefined: 2
+  no-unused-vars: 2
```
Comment (vcernomschi) [inline .eslintrc]: Please use the latest skeleton config

**Comment by devs-deep** [comment]
:white_check_mark: coverage increased from 0% to 45.23%

**Pull request merged**